### PR TITLE
bpo-39453: Add testcase for bpo-39453

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue39453

Automerge-Triggered-By: @pablogsal

Automerge-Triggered-By: @pablogsal

### CODE REVIEW

**Comment by corona10** [comment]
@pablogsal I will include this PR into backporting patch for https://github.com/python/cpython/pull/18181 to 3.8 3.7. 3.6

This case should be added for 3.8, 3.7 and 3.6

**Review by pablogsal — APPROVED** [review]

**Comment by pablogsal** [comment]
Thanks @corona10 ! 🎉

File: Objects/listobject.c [hunk 1]
```diff
@@ -452,7 +452,7 @@ list_contains(PyListObject *a, PyObject *el)
     for (i = 0, cmp = 0 ; cmp == 0 && i < Py_SIZE(a); ++i) {
         item = PyList_GET_ITEM(a, i);
         Py_INCREF(item);
-        cmp = PyObject_RichCompareBool(PyList_GET_ITEM(a, i), el, Py_EQ);
+        cmp = PyObject_RichCompareBool(item, el, Py_EQ);
```
Comment (corona10) [inline Objects/listobject.c:455]: @pablogsal 
Sorry, I found nit thing when I work on creating a backported patch.
I applied this nit fix on 3.8 backport. https://github.com/python/cpython/pull/18204

Please re-review for it :)
Thank you!

**Review by pablogsal — APPROVED** [review]

File: Objects/listobject.c (hunk 1 above)
Comment (pablogsal) [reply Objects/listobject.c:455]: Good catch, thanks!

**Pull request merged**